delivery_tnt_oca/models/tnt_request.py

```diff
@@ -39,7 +39,7 @@ def _send_api_request(
         if data is None:
             data = {}
         try:
-            headers = {"Content-Type": content_type}
+            headers = {"Content-Type": content_type, "charset": "UTF-8"}
             if auth:
                 headers["Authorization"] = "Basic {}".format(self.authorization)
             res = requests.post(url=url, data=data, headers=headers, timeout=60)
```

Comment:
But you still requires to encode `data`:

```suggestion
            res = requests.post(url=url, data=data.encode("utf-8"), headers=headers, timeout=60)
```